docs/guides/fetch-data/01-read-profile-data.md

```diff
@@ -0,0 +1,520 @@
+3. Read the Dataset
+   1. Fetch the profile's metadata
+   2. Fetch the profile's picture properties
+   3. Fetch the profile's universal receiver
+
+## Step 1 - Check for valid Blockchain Address
+
+When receiving data from a Universal Profile within an application, it is commonly used as input. You want to make sure that the address is valid in the first place. You can do a checkup by converting the value into a checksum address from the `web3.js` library. We need to install the library first. You can do this by opening up the terminal in the project`s folder of your choice.
+
+```shell
+npm install web3
+```
+
+After installation, we can move on with the JavaScript file. During the process, we will always use the same file. To make the guide more understandable, we use a sample profile address. You will most likely exchange this static variable with a dynamic value from an input field or fetching process within your app.
+
+```javascript title="read_profile.js"
+// Import and Network Setup
+const Web3 = require('web3');
+const web3 = new Web3('https://rpc.l14.lukso.network');
+
+// Our static Universal Profile address
+const sampleProfileAddress = '0x0C03fBa782b07bCf810DEb3b7f0595024A444F4e';
+
+/*
+ * Check if @param is a valid blockchain address
+ * @returns: boolean
+ */
+function isAddrValid(address) {
+  let formattedAddress = web3.utils.toChecksumAddress(address);
+  return web3.utils.checkAddressChecksum(formattedAddress);
+}
+
+// Debug
+console.log(isAddrValid(sampleProfileAddress));
+```
+
+If the function `isAddrValid` gives us back `true`, the address is valid and can be checked for contact data.
+
+## Step 2 - Call the Universal Profile
+
+To inspect the address and check if it has an ERC725 contract, we can call its interface through the `erc725.js` library. The instance of the contract will need the following information:
+
+- Schema which describes contract functions
+  - SupportedStandards will fetch the interface
+  - LSP3Profile fetches the data of the profile
+  - LSP1UniversalReceiverDelegate will fetch received assets
+- Provider has the RPC address of the called network
+- Address has the related blockchain address of the contract
+- Config has the base URL of the IPFS server
+
+Besides the schema, we also use `isomorphic-fetch` to fetch the HTTP response from the profile while using `node` for execution. You may not need this library if you use extended environments like `ReactJS` or `VueJS`. First, we have to install the libraries.
+
+```shell
+npm install @erc725/erc725.js
+npm install isomorphic-fetch
+```
+
+Within the JaveScript file, we can add this code snippet right underneath. After we embed the ERC725 object and the fetch functionality, we can declare all data needed to instantiate the ERC725 contract instance. As discussed above, our schema, provider, and network configuration need to be declared. While fetching, we can use the function from the previous step to handle errors that might occur.
```

Comment:
```suggestion
After we embed the ERC725 object and the fetch functionality, we can declare all data needed to instantiate the ERC725 contract instance. While fetching, we can use the function from the previous step to handle errors that might occur.
```

"we embed" ? You mean "we install" ?

ERC725 -> `erc725.js`

Reply:
I will use the term "import"